app/models/state_file_md_intake.rb

```diff
@@ -268,4 +268,8 @@ def has_banking_information_in_financial_resolution?
   def filer_disabled?
     primary_disabled_yes? || spouse_disabled_yes?
   end
+
+  def could_qualify_for_pension_exclusion?
```

Comment:
[Dust] this method name is kind of confusing, b/c I think if both filers are over 65, they could qualify for pension exclusion, correct?

maybe the more correct name is something like:
`should_warn_about_pension_exclusion?`